internal/server/kubeapi_handler.go

```diff
@@ -277,3 +277,20 @@ func (s *Server) cleanupUnusedHttpClients() {
 		return true
 	})
 }
+
+func (s *Server) checkHttpClientsConnection() {
+	log.Debug("checking health of http clients")
+	clients.Range(func(key, value any) bool {
+		clientName := key.(string)
+		tunnelId := strings.Split(clientName, "/")[0]
+
+		log.Debugf("checking health of client %s for tunnel %s", clientName, tunnelId)
+		err := s.kubeclient.UpdateConnectionProbe(tunnelId, s.remotedialer.HasSession(tunnelId))
```

Comment:
I think there may be scalability issues here.  We are updating all N ClusterConnects every minute.  At 1000 clusters that is almost 20 K8S updates / second, which seems like a lot.

I'm not sure what to do about this.  Maybe we need to make the probe interval configurable and document how to tune it for larger #s of clusters.  Or else find an alternative implementation, e.g., using per-session events to update the connection status.